mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_position.test

```diff
@@ -0,0 +1,20 @@
+--echo
+--echo # MDEV-27037 mysqlbinlog emits a warning when reaching EOF before stop-condition
+--echo
+
+--let assert_file= $MYSQLTEST_VARDIR/tmp/warn_pos_test_file.out
+--let data_file= $MYSQLTEST_VARDIR/std_data/master-bin.000001
+
+--echo Case: Stop position before EOF, WARNING must not appear
```

Comment:
For the `--echo`'s can you prefix the output with some indicator to show the origin is from an echo? `#` is pretty common (I think because there was an original idea of being able to use the `.result` file as input to `mariadbd`, but I've never done it myself or seen anyone actually do that), but I don't think it matters much as long as it is clear. It makes it easier to distinguish when reading the `.result` file.

Reply:
Definitely, I'll prepend it with `#` since that's what's common in the codebase already.